AmmoDriver.md

```diff
@@ -25,12 +25,12 @@ The Ammo.js driver provides many features and new functionality that the existin
 
 ## Installation
 
-Initial installation is the same as for Cannon.js. See: [Scripts](https://github.com/donmccurdy/aframe-physics-system/blob/master/README.md#installation), then see [Including the Ammo.js Build](#including-the-ammojs-build).
+Initial installation is the same as for Cannon.js. See: [Scripts](README.md#installation), then see [Including the Ammo.js Build](#including-the-ammojs-build).
```

Comment:
I'm not sure if the Scripts link will work once it's deployed to ghpages. We'll need to verify this link once merged.

Reply:
The link from README.md to AmmoDriver.md works on ghpages, so this ought to as well.